### PR TITLE
Only show the entries from the current user

### LINKED ISSUE #7: Mostrar apenas as `entradas` do usuário corrente.
photo5170656620680685643

### PR DESCRIPTION
**Pull request by renanbona**
This PR closes #7 

- Change index and create action from EntriesController
- Add devise test helpers
- Change entries controller spec
- Add entries request spec
- Remove unnecessary comments

### CODE REVIEW

File: spec/controllers/entries_controller_spec.rb [hunk 1]
```diff
@@ -37,86 +11,104 @@
...
-        post :create, params: { entry: valid_attributes }, session: valid_session
+        post :create, params: { entry: valid_attributes }
         expect(flash[:success]).to match(/Entry was successfully created./)
       end
     end
 
     context "with invalid params" do
-      it "returns a failure response (i.e. to display the 'new' template)" do
-        post :create, params: { entry: invalid_attributes }, session: valid_session
+      xit "returns a failure response (i.e. to display the 'new' template)" do
```
Comment (renatamarques97) [inline spec/controllers/entries_controller_spec.rb]: why did you add an x to these specs? it weren't failure before

Comment (renanbona) [reply spec/controllers/entries_controller_spec.rb]: Now the entry is created with the current user id, so this feature doesn't have the context with invalid attributes. The entry doesn't have required attributes yet.

Comment (renatamarques97) [reply spec/controllers/entries_controller_spec.rb]: right, i get it

Comment (marclerodrigues) [reply spec/controllers/entries_controller_spec.rb]: Delete the specs that you're skipping, skipped specs tend to become garbage in the project. If we need it in the future we will add them.

**Pull request merged**